definitions/artifacts/mongo-authentication.json

```diff
@@ -20,7 +20,17 @@
       ],
       "properties": {
         "infrastructure": {
-          "$ref": "../types/k8s-application-infrastructure.json"
+          "title": "Mongo Cluster Infrastructure",
+          "description": "Mongo cluster infrastructure configuration.",
+          "type": "object",
+          "oneOf": [
+            {
+              "$ref": "../types/k8s-application-infrastructure.json"
+            },
+            {
+              "$ref": "../types/azure-resource-id.json"
+            }
```

Comment:
```suggestion
            {
              "ari": {
                "$ref": "../types/azure-resource-id.json"
              }
            }
```

Reply:
Small change, but we need to nest the resource id ref under the key we want, in this case `ari`. 

Tested w/ the schema validator

<img width="1373" alt="Screenshot 2022-07-20 at 4 23 27 PM" src="https://user-images.githubusercontent.com/651833/180099124-666c09d7-7453-4cd0-9859-b5596a8163ee.png">

Reply:
<img width="1454" alt="Screenshot 2022-07-20 at 4 26 19 PM" src="https://user-images.githubusercontent.com/651833/180099362-adb0bede-f789-46f0-bf08-2b5c812683c4.png">